### PR TITLE
Fix #65 provide environment in the get cmd

### LINKED ISSUE #65: Environment id is not supported for getting a content type
## Expected Behavior

When requesting a content type and specifying the space and the content_type should be retrieved from that space and content type, e.g:

`contentful --space-id foospace --environment-id barvironment content-type get --id bazcontenttype`

## Actual Behavior

The environment id is ignored and the master id is selected

## Possible Solution

Allow specifying the `--environment` argument here:
https://github.com/contentful/contentful-cli/blob/56482f91f6a1d18108f9be8ec93322f349c40553/lib/cmds/content-type_cmds/get.js#L17

-   **Package Version**: Contentful CLI 0.14.2

### PR DESCRIPTION
**Pull request by M3kH**
## Summary

Introduce the `environmentId` to the `get` command. 

## Description

Solves what describe in #65

### CODE REVIEW

**Comment by axe312ger** [comment]
Thank you. LGTM, just waiting for CI.

**Comment by M3kH** [comment]
Seems that your test suite is broken since you move from `ava` to `jest`.
Not sure how I can help there.

File: lib/cmds/content-type_cmds/get.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const builder = (yargs) => {
   return yargs
     .option('id', { type: 'string', demand: true, describe: 'Content Type id' })
     .option('space-id', { type: 'string', describe: 'Space id' })
+    .option('environment-id', { type: 'string', describe: 'Environment id' })
```
Comment (Khaledgarbaya) [inline lib/cmds/content-type_cmds/get.js]: Can you add `default: 'master'` here to make sure that we always fallback to master

Comment (M3kH) [reply lib/cmds/content-type_cmds/get.js]: 👍

Comment (Khaledgarbaya) [reply lib/cmds/content-type_cmds/get.js]: I will take care of the unit tests

Comment (M3kH) [reply lib/cmds/content-type_cmds/get.js]: Fixed.

**Comment by Khaledgarbaya** [comment]
Thanks @M3kH I will take it from here and fix the unit test.
I will let you know when we'll release a new version

**Comment by Khaledgarbaya** [comment]
I ran the tests locally and all is good, I will this for now

**Pull request merged**